.github/CODEOWNERS

```diff
@@ -2,20 +2,20 @@
 # https://help.github.com/en/articles/about-code-owners
 
 # Restricted Paths
-*                                        @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads
-/.github/workflows                       @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @ijjk
-/packages/fs-detectors                   @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @agadzik @chloetedder
-/packages/next                           @timneutkens @ijjk @ztanner @huozhi
-/packages/routing-utils                  @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @ijjk
-/packages/static-build                   @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads
+*                                        @vercel/ci-cd
+/.github/workflows                       @vercel/ci-cd @ijjk
+/packages/fs-detectors                   @vercel/ci-cd @agadzik @chloetedder
+/packages/next                           @vercel/ci-cd @timneutkens @ijjk @ztanner @huozhi
+/packages/routing-utils                  @vercel/ci-cd @ijjk
+/packages/static-build                   @vercel/ci-cd
 /packages/edge                           @vercel/compute
 /packages/functions                      @vercel/compute
 /packages/firewall                       @cramforce @sueplex
-/examples                                @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @leerob
-/examples/create-react-app               @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Timer
-/examples/nextjs                         @ijjk @ztanner @huozhi
-/packages/node                           @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Kikobeats
-/packages/related-projects               @TooTallNate @EndangeredMassa @trek @onsclom @jeffsee55 @erikareads @Kikobeats @tknickman @mknichel
+/examples                                @vercel/ci-cd @leerob
+/examples/create-react-app               @vercel/ci-cd @Timer
+/examples/nextjs                         @vercel/ci-cd @ijjk @ztanner @huozhi
+/packages/node                           @vercel/ci-cd @Kikobeats
+/packages/related-projects               @vercel/ci-cd @Kikobeats @mknichel
```

Comment:
We intentionally do not use github teams for ownership because it removes the review request from others after a single person in that team reviews the PR. If you use Github's view of currently requested PRs to know what needs your review, you could never see a subset of the PRs that came through. Leaving it written out means you were always requested.

This group is also broader than the original list. This means notifications will be sent to far more people, many of which have little context in this repo and do not have ownership over its changes. If we do switch this a github team, I think it should be a smaller group.

Reply:
I would argue that GitHub's default behaviour for notifications is better - as it can lead to notification fatigue if folks are seeing notifications for PRs which have already been merged. However we'll look to migrate this to Vercel Code Owners next, which will give us more fine grained control over the notification logic.

At this time we're ramping up everyone in CICD to be owners of this repository. Given that the agent team are mostly in Europe and currently all the global approves for this repository are in the USA we need to make sure that we're productive to ship!